src/codex_plus/llm_execution_middleware.py

```
@@ -379,42 +383,72 @@ async def process_request(self, request, path: str):
+                try:
+                    response = session.request(
+                        request.method,
+                        target_url,
+                        headers=clean_headers,
+                        data=body if body else None,
+                        stream=True,
+                        timeout=30
+                    )
+                    break
+                except requests.exceptions.RequestException as exc:
+                    last_exception = exc
```

Ruff (0.13.3)

Local variable `last_exception` is assigned to but never used

Remove assignment to unused variable `last_exception`

(F841)